internal/provider/resource_apply_module.go

```diff
@@ -406,21 +477,48 @@ func (t *applyModuleResource) doRun(ctx context.Context,
 	// while plannedRun.Plan.Status is "finished".
 	//
 	if !strings.HasPrefix(string(plannedRun.Status), "planned") {
```

Comment:
These checks can be updated to look for the exact status instead of the prefix.

Reply:
Done.